stacks-signer/src/config.rs

```diff
@@ -39,6 +39,7 @@ const BLOCK_PROPOSAL_VALIDATION_TIMEOUT_MS: u64 = 120_000;
 const DEFAULT_FIRST_PROPOSAL_BURN_BLOCK_TIMING_SECS: u64 = 60;
 const DEFAULT_TENURE_LAST_BLOCK_PROPOSAL_TIMEOUT_SECS: u64 = 30;
 const TENURE_IDLE_TIMEOUT_SECS: u64 = 300;
+const DEFAULT_BLOCK_PROPOSAL_MAX_AGE_SECS: u64 = 600;
```

Comment:
This is a long time. Are there consequences to making it shorter -- i.e. a small integer multiple of the expected RTT for the peer network?

Reply:
I am totally fine with making it shorter. This was just pulled out of my butt ...

I am not sure what we would consider a good base...or even what the expected RTT is

Reply:
Made a ticket to look into a better default value. https://github.com/stacks-network/stacks-core/issues/5566